Package.swift

```diff
@@ -0,0 +1,101 @@
+// swift-tools-version: 5.6
+//   The swift-tools-version declares the minimum version of Swift required to build this package and MUST be the first
+//   line of this file. 5.6 is required to support zip files for the pod archive binaryTarget.
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+//
+// A user of the Swift Package Manager (SPM) package will consume this file directly from the ORT github repository.
+// For context, the end user's config will look something like:
+//
+//     dependencies: [
+//       .package(url: "https://github.com/microsoft/onnxruntime", branch: "rel-1.15.0"),
+//       ...
+//     ],
+//
+// NOTE: The direct consumption creates a somewhat complicated setup to 'release' a new version of the ORT SPM package.
+//       TBD: how to manage the release process
+
+import PackageDescription
+import class Foundation.ProcessInfo
+
+let package = Package(
+    name: "onnxruntime",
+    platforms: [.iOS(.v11)],
+    products: [
+        .library(name: "onnxruntime",
+                 type: .static,
+                 targets: ["OnnxRuntimeBindings"]),
+    ],
+    dependencies: [],
+    targets: [
+        .target(name: "OnnxRuntimeBindings",
+                dependencies: ["onnxruntime"],
+                path: "objectivec",
+                exclude: ["test", "docs"],
+                cxxSettings: [
+                    .unsafeFlags(["-std=c++17",
+                                  "-fobjc-arc-exceptions"
+                                 ]),
+                ], linkerSettings: [
+                    .unsafeFlags(["-ObjC"]),
+                ]),
+        .testTarget(name: "OnnxRuntimeBindingsTests",
+                    dependencies: ["OnnxRuntimeBindings"],
+                    path: "swift/OnnxRuntimeBindingsTests",
+                    resources: [
+                        .copy("Resources/single_add.basic.ort")
+                    ]),
+    ]
+)
+
+// Add the ORT iOS Pod archive as a binary target.
+//
+// There are 3 scenarios:
+//
+// Release branch/tag of ORT github repo:
+//    Target will be released pod archive and its checksum.
+//
+// `main` branch of ORT github repo:
+//    Invalid. We do not have a pod archive that is guaranteed to work with the latest code on main as the objective-c
+//    bindings may have changed.
+//
+// CI or local testing where you have built/obtained the iOS Pod archive matching the current source code.
+//    Requires the ORT_IOS_POD_LOCAL_PATH environment variable to be set to specify the location of the pod.
+if ProcessInfo.processInfo.environment["ORT_IOS_POD_LOCAL_PATH"] != nil {
```

Comment:
```suggestion
if let pod_archive_path = ProcessInfo.processInfo.environment["ORT_IOS_POD_LOCAL_PATH"] {
```
see https://docs.swift.org/swift-book/documentation/the-swift-programming-language/thebasics#Optional-Binding